### ISSUE TITLE
README's Install as Usual Line Seems Wrong?

### ISSUE THREAD

**Issue by McFateM** [digitalutsc/islandora_solution_pack_oralhistories#123]
The first line of the README.md file's "Installation" section reads "Install as usual, see this for further information."   But that doesn't seem right to me.  Unless I'm mistaken, the linked instructions there apply only to Drupal modules that have been contributed to and accepted by Drupal.org.  Or am I missing something here?

FYI... I am writing a bit of ISLE documentation that covers installing this module using Drush inside an ISLE (Dockerized Islandora) container.  I'd be happy to push that docmentation back here in a PR if you are interested in reading and maybe adopting it.

**Comment by MarcusBarnes** [comment]
@McFateM That line in the README was an attempt to mirror the [Islandora README template](https://github.com/Islandora/islandora/wiki/Readme-template). We can certainly explore improving this section of the README.

**Comment by McFateM** [comment]
@MarcusBarnes Hello Marcus and thanks for the quick response.  I figured that language came from the Islandora template, but I was nonetheless disappointed that the link leads to a **very lengthy** explanation of ways to download, enable and configure a typical Drupal 7 module.  I'm just not confident there's anything in the document addressing the "typical" means of installing an Islandora-Labs module, a process that generally involves 'git clone' followed by some Drush commands and follow-up. 

I could be wrong?  If so, someone please enlighten me.  Thanks!

**Comment by MarcusBarnes** [comment]
@McFateM No, I don't think you're wrong.  The section can be improved.  Maybe you can submit your suggested improvements as a PR based on the documentation you're creating and we can go from there?  

On a different note, will your documentation be publicly available?  If so, we can explore whether it would be useful linking to it in the Wiki or elsewhere.

**Comment by McFateM** [comment]
Yes and yes.  I hope that my documentation will soon become part of https://github.com/Islandora-Collaboration-Group/ISLE-Documentation, and if/when it does it will be publicly available.  If it plays well there perhaps I will submit a PR to adopt it here too.

**Comment by MarcusBarnes** [comment]
@McFateM Would you be able to provide a PR for the documentation enhancements you were working on?  It would be great to include your suggested improvements as appropriate as part of the work for the upcoming release of the module.  Thank you in advance!

**Comment by McFateM** [comment]
I'm really swamped with work right now @MarcusBarnes, and ISLE is being re-factored so I'm not sure where my documentation work fits into the new scheme.  I have an ISLE meeting next Wednesday and will bring up the issue then if I can.  I also hope to be testing a new ISLE pre-release candidate by that time and will include oral histories in that test.  So, I expect more changes/documentation to come, but not sure I can prep anything for at least another two weeks or more.
